### DISCUSSION TITLE
Images hosted in Github Repo not displaying

### DISCUSSION THREAD

**Discussion by sohamdasdesigns** [community/community#83145] [Programming Help]
### Body

![Screenshot from 2023-12-30 14-52-37](https://github.com/community/community/assets/88555443/84a15c12-d7c1-46d5-8d5e-47dd1eec16f7)

The uploaded png/jpg images are not displaying in the github Repo. Even though they are existing in the repository when downloaded. Can someone suggest any solution?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by faridnec** [comment]
Hey I just checked your repo and it shows the image. Maybe u can check the internet connection?

**Comment by sohamdasdesigns** [comment]
Yes.. It is showing , maybe it occurred due to the internet connection .. Thank you..
